TACC/Core-CMS-Custom
# feat/fix(apcd): support isolated local CMS database and volumes

## Overview

Provide instruction and file to support an isolated APCD database (rather than sharing `core_cms_postgres`).

<details>

If developer already has a [TACC/Core-Portal](https://github.com/TACC/Core-Portal), and uses `docker-compose.dev.yml`, then they may experience a conflict error for `core_cms_postgres` container/host/volume name.

If developers work on multiple [TACC/Core-CMS](https://github.com/TACC/Core-CMS)-based projects, and they build more than one, they may encounter a conflict error for `core_cms_postgres` container/host/volume name.

If developer works on multiple [TACC/Core-CMS](https://github.com/TACC/Core-CMS)-based projects, and they clone remote database to local database (for accurate testing), they would overwrite the databases other local instances of CMS projects are using.

</details>

## Related

- affects **current** developers locally testing APCD *and* a [TACC/Core-CMS](https://github.com/TACC/Core-CMS)-based project
- affects **new** developers locally testing APCD *and* [TACC/Core-Portal](https://github.com/TACC/Core-Portal)
- affects **future** developers locally testing APCD *and* a future CMS project in this repo

## Changes

- add alternative docker-compose file
- document usage of that file

## Testing

1. Have a local [TACC/Core-Portal](https://github.com/TACC/Core-Portal) instance.
2. Run a local `apcd-cms` instance __using the original docker compose dev file__.
3. ⓧ Verify conflict error.
2. Run a local `apcd-cms` instance __using the new alternative docker compose dev file__.
4. ✓ Verify __no__ conflict error.

## UI

N/A

## Notes

Alternatively, I could have a sample such file in root, and document how to clone and edit it per project as needed.
